docs/samples/customdomain/README.md

```diff
@@ -0,0 +1,134 @@
+# Setting up custom domain name
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. You have a custom domain configured to route incoming traffic to the cluster's Ingress gateway.
```

Comment:
Maybe expand this a bit more, if users can route their custom domain directly to the istio-ingressgateway's loadbalancer. Then we can skip the Ingress resource step.